hw/ip_templates/racl_ctrl/dv/racl_ctrl_sim_cfg.hjson.tpl

```diff
@@ -5,12 +5,18 @@
   // Fusesoc core file used for building the file list.
   fusesoc_core: ${instance_vlnv(f"lowrisc:dv:{module_instance_name}_sim:0.1")}
 
+  // Testplan hjson file. Note that this is not templated.
```

Comment:
This comment confused me. Maybe I am just caffeine-deprived, but could you elaborate a bit more please? Thanks :)

Reply:
Oops: that could definitely have been clearer!